test/data-engine.test.js

```diff
@@ -194,24 +194,24 @@ describe('DataEngine class', () => {
         data
       }
 
-      const list = document.createElement('ul')
+      const list = document.createElement('ol')
       list.innerHTML =
```

Comment:
would backticks be better here so you don't have to keep writing `+` after each line?

```
list.innerHtml = `
<li data-id="1">One
  <ol data-id="1">
```

?

Reply:
Not really. I tried it before. Backticks cause some white spaces which make the tests fail, unfortunately.

Reply:
ah good point